app/views/responses/form.html.erb

```diff
@@ -2,7 +2,7 @@
 <% content_for(:per_page_js){ javascript_google_maps } %>
 
 <%# title includes ID if defined %>
-<% @title_args = {:id => @response.id} %>
+<% @title_args = {id: @response.shortcode} %>
```

Comment:
Also I think we can get rid of the `#` in front of the shortcode in the title now as it doesn't really make sense anymore.